web/src/analytics_test_helpers.ts

```diff
@@ -1,6 +1,16 @@
 import fetchMock from "jest-fetch-mock"
 import { Tags } from "./analytics"
 
+// at some point we might want to be clever about this and only mock the responses of calls to /api/analytics
```

Comment:
seems like this is likely to bite us, and the fix for it would be shorter than this comment?
https://www.npmjs.com/package/jest-fetch-mock#conditional-mocking
tho i could be missing something

Reply:
it looks like jest-fetch-mock's behavior with multiple calls is not great, so I've switched our tests to fetch-mock, which feels a lot nicer! Discussed further in [another comment](https://github.com/tilt-dev/tilt/pull/4415#issuecomment-817915482)